modules/govuk_ci/manifests/agent.pp

```diff
@@ -11,4 +11,16 @@
   include ::govuk_ci::agent::postgresql
   include ::govuk_ci::agent::mysql
 
+  ufw::allow { 'allow-jenkins-slave-swarm-to-listen-on-ephemeral-ports':
+    port  => '32768:65535',
+    proto => 'udp',
+    ip    => 'any',
+  }
```

Comment:
Do we have the swarm plugin installed yet? Shouldn't we install the plugin and add a firewall rule for it at the same time? Seems like this would be better done as part of [this story](https://trello.com/c/sH8ZfvdZ/294-connect-a-jenkins-agent-to-the-new-jenkins-master) so that we can keep track of what's going on.